### ISSUE TITLE
Wrong image version of jaeger-collector and jaeger-agent being pulled

### ISSUE THREAD

**Issue by kevchu3** [istio/istio#16456]
**Bug description**
The istio-operator pod will deploy the jaeger-collector and jaeger-query deployments.  Based on the OpenShift docs [1], we should be able to use tag version 1.13.1 based on the example below:
```
  tracing:
      enabled: false
      jaeger:
        tag: 1.13.1
        template: all-in-one
        agentStrategy: DaemonSet
```

But when I deploy the jaeger-operator which in turn deploys the jaeger-collector and jaeger-query pods, I see that the wrong image version is used.  Unfortunately, based on the registry.redhat.io container catalog, tag version 1.12.0 doesn't exist [2], which results in an ImagePullBackoff error when spinning up those pods.

Result of `oc describe pod jaeger-collector`:
```
Containers:
  jaeger-collector:
    Container ID:  
    Image:         registry.redhat.io/distributed-tracing-tech-preview/jaeger-collector:1.12.0
```

Result of `oc describe pod jaeger-agent`:
```
  jaeger-agent:
    Container ID:  
    Image:         registry.redhat.io/distributed-tracing-tech-preview/jaeger-agent:1.12.0
```

**Affected product area (please put an X in all that apply)**

[X] Configuration Infrastructure
[ ] Docs
[X] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

**Expected behavior**
The specified image tag version (in this case, 1.13.1) should really be pulled, not the default tag 1.12.0 instead (which coincidentally points to a non-existent tag version of the image).

**Steps to reproduce the bug**
These steps are taken from the install docs [3] [4]:
```
$ oc new-project observability # create the project for the jaeger operator
$ oc create -n observability -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/v1.13.1/deploy/crds/jaegertracing_v1_jaeger_crd.yaml
$ oc create -n observability -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/v1.13.1/deploy/service_account.yaml
$ oc create -n observability -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/v1.13.1/deploy/role.yaml
$ oc create -n observability -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/v1.13.1/deploy/role_binding.yaml
$ oc create -n observability -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/v1.13.1/deploy/operator.yaml

$ oc new-project istio-operator
$ oc new-project istio-system
$ oc apply -n istio-operator -f https://raw.githubusercontent.com/Maistra/istio-operator/maistra-0.12/deploy/servicemesh-operator.yaml
```

**Version (include the output of `istioctl version --remote` and `kubectl version`)**
OpenShift 3.11
Service Mesh / Istio 0.12
Jaeger 1.13.1

**How was Istio installed?**
See steps to reproduce the bug above

**Environment where bug was observed (cloud vendor, OS, etc)**
On-prem lab cluster

**Comment by brian-avery** [comment]
Hi @kevchu3

Thanks for opening this issue. We've been tracking it in our Jira here: https://issues.jboss.org/browse/MAISTRA-684?jql=project%20%3D%20MAISTRA%20AND%20text%20~%20%22Jaeger%20image%22. The documentation has a notice concerning this it will be fixed in the next release.

**Comment by brian-avery** [comment]
Note that this bug affects neither the upstream Istio installer nor upstream Istio configuration infrastructure and is specific to Maistra.

**Comment by kevchu3** [comment]
Hi @brian-avery, thanks for correlating this with a tracked issue in Jira.

I thought to open this in Maistra here (https://github.com/Maistra/istio-operator) but their repo doesn't allow us to open new issues unfortunately (not the Istio repo's problem), so I defaulted back here.

**Comment by brian-avery** [comment]
Hi @kevchu3 We just released the 1.0.0 version of ServiceMesh which fixes this issue. Would you be able to upgrade? Also, we usually use Jira (https://issues.jboss.org/projects/MAISTRA) for Maistra/ServiceMesh issues.

**Comment by kevchu3** [comment]
@brian-avery I'm currently running ServiceMesh on an OpenShift 3.11 cluster, so it is Tech Preview and thus will not be able to use the 1.0.0 version.  Nevertheless, I'm able to work around by specifying the version in the deployment to pull the correct image.  I'll go ahead and close out this issue as you mentioned that the GA
